src/main/java/apoc/custom/CypherProcedures.java

```diff
@@ -589,7 +602,7 @@ public void unavailable() {
         }
 
         public static Map<String, Object> storeFunction(GraphDatabaseAPI api, UserFunctionSignature signature, String statement, boolean forceSingle) {
-            Map<String, Object> data = map("statement", statement, "forceSingle", forceSingle, "signature", signature.toString(), "description", signature.description());
+            Map<String, Object> data = map("statement", statement, "forceSingle", forceSingle, "signature", signature.toString(), "description", signature.description().orElse(""));
```

Comment:
sorry I overlooked this b/c it was too far on the right :) can you use `.orElse(null)` so that we're consistent? in the 2 places?

Reply:
:+1: